src/main/java/list.java

```diff
@@ -0,0 +1,42 @@
+public class List {
+    private static String[] currentList = new String[100];
+    public static int listSize = 0;
+    private static int[] isDone = new int[100];
```

Comment:
name of this array sounds like a boolean name. 
maybe change the name to better represent what you are trying to store